airbyte-webapp-e2e-tests/cypress/pages/createConnectorPage.ts

```diff
@@ -6,7 +6,7 @@ const databaseInput = "input[name='connectionConfiguration.database']";
 const usernameInput = "input[name='connectionConfiguration.username']";
 const passwordInput = "input[name='connectionConfiguration.password']";
 const pokemonNameInput = "input[name='connectionConfiguration.pokemon_name']";
-const schemaInput = "input[name='connectionConfiguration.schemas']";
+const schemaInput = "#react-select-5-input";
```

Comment:
Double-check the ID, it looks like the generated one

Reply:
fixed, thanks!